inference/core/cache/transforms.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union
+
+from fastapi.encoders import jsonable_encoder
+
+from inference.core.devices.utils import GLOBAL_INFERENCE_SERVER_ID
+from inference.core.entities.requests.inference import InferenceRequest
+from inference.core.entities.responses.inference import InferenceResponse
+from inference.core.env import CONDENSED_CACHED
+from inference.core.logger import logger
+from inference.core.version import __version__
+
+
+def to_cachable_inference_item(
+    infer_request: InferenceRequest,
+    infer_response: Union[InferenceResponse, list[InferenceResponse]],
+) -> dict:
+    request = jsonable_encoder(infer_request.dict())
```

Comment:
I think we can exclude fields which might make this line run faster. So maybe, something like:
```suggestion
    excluded_fields = ['image'] if CONDENSED_CACHE else []
    request = jsonable_encoder(infer_request.dict(exclude=excluded_fields))
```

Docs for `.dict(...)`: https://docs.pydantic.dev/1.10/usage/exporting_models/